README.md

```diff
@@ -52,6 +52,7 @@ Starting point was `spm_BIDS.m` from [SPM12](https://github.com/spm/spm12) ([doc
 ## Other tools (MATLAB only)
 - [dicm2nii](https://github.com/xiangruili/dicm2nii): A DICOM to BIDS converter
 - [imtool3D_BIDS](https://github.com/tanguyduval/imtool3D_td): A 3D viewer for BIDS directory
+- [Brainstorm](https://github.com/brainstorm-tools/brainstorm3): Comprehensive brain analysis toolbox (includes BIDS import functionality)
```

Comment:
```suggestion
- [Brainstorm](https://github.com/brainstorm-tools/brainstorm3): Comprehensive brain analysis toolbox (includes BIDS [import](https://neuroimage.usc.edu/brainstorm/Tutorials/RestingOmega#BIDS_specifications) and [export](https://neuroimage.usc.edu/brainstorm/ExportBids) functionality)
```

Just a suggestion to have the link in there directly to some import and export sections. 

What do you think?

Reply:
maybe @martcous you could update the BIDS tutorial to include both import and export? The import part should be quite straight forward.

And the we can update the text like this:

```suggestion
- [Brainstorm](https://github.com/brainstorm-tools/brainstorm3): Comprehensive brain analysis toolbox (includes BIDS [import and export] (https://neuroimage.usc.edu/brainstorm/ExportBids) and different examples dealing with BIDS datasets (e.g. [group analysis from a MEG visual dataset](https://neuroimage.usc.edu/brainstorm/Tutorials/VisualGroup, [resting state analysis from OMEGA datasets](https://neuroimage.usc.edu/brainstorm/Tutorials/RestingOmega#BIDS_specifications) )
```

Reply:
Oh I had not seen your suggestions @guiomar  and thumb up @cMadan.

Will update and merge now. Sorry for being slow on this.

Reply:
Sorry, maybe I placed them in the wrong place. Thanks for taking the time to include them :)

Reply:
no worries